src/dramatiq_abort/backend.py

```diff
@@ -36,8 +36,8 @@ def poll(self, key: bytes) -> bool:  # pragma: no cover
         raise NotImplementedError
 
     @abc.abstractmethod
-    def notify(self, key: bytes, ttl: int) -> None:  # pragma: no cover
-        """Signal an event.
+    def notify(self, keys: Iterable[bytes], ttl: int) -> None:  # pragma: no cover
```

Comment:
Why are you changing this? Looking at the implementation, `notify` appears to be the same as `notify_many`.

Reply:
I would suggest to get rid of `notify_many` and only keep `notify` with your changes. Also, please update the docstring for `key`!